app/pages/home-for-user/circle-ribbon.jsx

```diff
@@ -170,15 +181,17 @@ const CircleRibbon = React.createClass({
           </defs>
 
           {!!this.props.image && (
-            <image
-              xlinkHref={this.props.image}
-              x={this.props.weight + this.props.gap}
-              y={this.props.weight + this.props.gap}
-              width={imageSize}
-              height={imageSize}
-              clipPath={`url('#circle-ribbon-clip-${this.id}')`}
-              className={`url('#circle-ribbon-shadow-${this.id}')`}
-            />
+            <Link to={`/users/${this.props.user.login}/stats`}>
```

Comment:
Since this is in a SVG, then I think you'll want to be using the `SVGLink` component here. Also it needs an aria label.